app/Platform/Services/GameSuggestions/Strategies/SharedHubStrategy.php

```diff
@@ -36,6 +36,8 @@ public function select(): ?Game
             ->whereHas('games', function ($query) {
                 $query->where('game_id', '!=', $this->sourceGame->id); // needs other games too!
             })
+            ->where('Title', 'NOT LIKE', "%Meta%")
+            ->where('Title', 'NOT LIKE', "%Misc%")
```

Comment:
This looks safe for now, but I'd recommend changing it in a similar manner.

Reply:
Updated to "%Misc. - %" in latest.